meta-refkit/conf/distro/refkit.conf

```diff
@@ -187,6 +187,19 @@ BBMASK += " \
     meta-intel/common/recipes-core/ovmf \
 "
 
+# Without this we end up taking TPM2.0 recipes from meta-security
+# instead of meta-measured. We only need meta-tpm because of swtpm.
+# If we move that to meta-measured, we could instead drop meta-tpm.
+BBMASK += "meta-security/meta-tpm/recipes-tpm/tpm2"
```

Comment:
With swtpm2 and others copied to meta-refkit, do we need meta-security at all? Dropping it could simplify ```BBMASK```ing.

Reply:
meta-security is still the place from where we get the TPM 1.2 recipes plus some other things, like keyutils.

Reply:
OK, at quick glance I did not find any other use for meta-security than meta-tpm (supported recipes says ```keyutils@security-framework```)